ci/publish.sh

```diff
@@ -1,16 +1,35 @@
 #!/bin/bash
 set -eo pipefail
+
 retval=0
+projects_to_publish=('sdk' 'tools/sctool' 'tools/sidechains-sdk-account_sctools')
```

Comment:
As discussed with @paolocappelletti , we should publish also `tools/sidechains-sdk-utxo_sctools`